hw3/src/MyListImpl.java

```diff
@@ -0,0 +1,143 @@
+import java.util.*;
+
+public class MyListImpl implements MyList {
+
+    private Person[] arr = new Person[4];
+
+    @Override
+    public int size() {
+        return arr.length;
+    }
+
+    @Override
+    public boolean isEmpty() {
+        return arr.length == 0;
+    }
+
+    @Override
+    public boolean contains(Object o) {
+        if (o instanceof Person) {
+            for (Person p : arr) {
+                return p.equals(o);
+            }
+        }
+        return false;
+    }
+
+
+    @Override
+    public Iterator iterator() {
+        return null;
+    }
+
+    @Override
+    public Object[] toArray() {
+        return new Object[0];
+    }
+
+    @Override
+    public boolean add(Object o) {
+        arr = Arrays.copyOf(arr, arr.length + 1);
+        arr[arr.length - 1] = new Person();
+        return false;
```

Comment:
why "false"

Reply:
why not?  why the "add" method requires Boolean output?